server/src/main/java/org/opensearch/extensions/action/ExtensionsActionResponse.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.action;
+
+import org.opensearch.action.ActionResponse;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.xcontent.ToXContentObject;
+import org.opensearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+public class ExtensionsActionResponse extends ActionResponse implements ToXContentObject {
+    String myFirstResponse;
+
+    ExtensionsActionResponse(String myFirstResponse) {
+        this.myFirstResponse = myFirstResponse;
+    }
+
+    ExtensionsActionResponse(StreamInput in) throws IOException {
+        super(in);
+        myFirstResponse = in.readString();
+    }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        out.writeString(myFirstResponse);
```

Comment:
I haven't dug into the implementation details, but when I see a `super(in)` I expect to see a matching `super(out)`.  I know the super is required for transport requests but not responses, and in the latter case the super call is a noop.  

All that to say, either add a `super(out)` here or delete the `super(in)` above.